README.rst

```diff
@@ -0,0 +1,20 @@
+GooDoo
+======
+
+GooDoo (codename: BestJa) is an utility for professional management of
+volunteers’ work.
+
+There are two websites for which GooDoo has been created, owned by
+`Federacja Polskich Banków Żywności <http://bankizywnosci.pl/>`__ and
+`Uniwersyteckie Centrum Wolontariatu
+<http://wolontariat.uw.edu.pl/>`__. Modules related to these websites
+are marked with suffixes ``_fpbz`` and ``_ucw``, respectively.
```

Comment:
We could also add an information that modules that are not prefixed with "bestja_" may be used outside of the Bestja ecosystem, as a standalone modules in any Odoo installation.